Tiltfile

```diff
@@ -232,13 +232,7 @@ def enable_provider(name, debug):
     links = []
 
     if debug_port != 0:
-        # Add delve when debugging. Delve will always listen on the pod side on port 30000.
-        entrypoint = ["sh", "/start.sh", "/dlv", "--listen=:" + str(30000), "--accept-multiclient", "--api-version=2", "--headless=true", "exec", "--", "/manager"]
         port_forwards.append(port_forward(debug_port, 30000))
```

Comment:
Should me make this actually configurable?

Reply:
I think the "local" port has to be configurable as it is today. I think it's fine to hard-code the "server-side" port (as it is today)